### PR TITLE
Adding Files

### PR DESCRIPTION
**Pull request by zaidansari42**
Please checkout the details now, was having an issue with the previous PR

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  position: absolute;
+  content: "";
+  height: 200%;
+  width: 200%;
+  bottom: -50%;
+  left: -50%;
+  border-radius: 44%;
+  background-color: var(--background);
+  /* If we use forwards, then the animation will stop after completing */
+  /* If we use infinite, then it will keep on looping it. */
```
Comment (Rehan-Khan-Engineer) [inline styles.css:67]: I think you can remove this. It's a very general comment. @zaidansari42

**Review by Rehan-Khan-Engineer — APPROVED** [review]
LGTM

File: styles.css (hunk 1 above)
Comment (zaidansari42) [reply styles.css:67]: I just added those for my knowledge, It was my first time using this

**Comment by zaidansari42** [comment]
Thanks for the reviews evryone

**Pull request closed without merging**